README_JP.md

```diff
@@ -114,8 +114,6 @@
 
 # FAQ （更新中，ボランティア募集中）
 
-よくある質問，[クリック](./FAQ.md)
```

Comment:
Now the new url is here :https://community.wuhan2020.org.cn/ja-jp/docs/overview/faq.html